docs/docusaurus.config.ts

```diff
@@ -0,0 +1,157 @@
+      apiKey: 'd02cc0a6a61178b25da550212925226b',
+      indexName: 'frigate',
+    },
+    docs: {
+      sidebar: {
+        hideable: true,
+      },
+    },
+    prism: {
+      additionalLanguages: ['bash', 'json'],
+    },
+    languageTabs: [
+      {
+        highlight: 'python',
+        language: 'python',
+        logoClass: 'python',
+      },
+      {
+        highlight: 'javascript',
+        language: 'nodejs',
+        logoClass: 'nodejs',
+      },
+      {
+        highlight: 'javascript',
+        language: 'javascript',
+        logoClass: 'javascript',
+      },
+      {
+        highlight: 'bash',
+        language: 'curl',
+        logoClass: 'curl',
+      },
+      {
+        highlight: "rust",
+        language: "rust",
+        logoClass: "rust",
+      },
+    ],
+    navbar: {
+      title: 'Frigate',
+      logo: {
+        alt: 'Frigate',
+        src: 'img/logo.svg',
+        srcDark: 'img/logo-dark.svg',
+      },
+      items: [
+        {
+          to: '/',
+          activeBasePath: 'docs',
+          label: 'Docs',
+          position: 'left',
+        },
+        {
+          href: 'https://frigate.video',
+          label: 'Website',
+          position: 'right',
+        },
+        {
+          href: 'http://demo.frigate.video',
+          label: 'Demo',
+          position: 'right',
+        },
+        {
+          href: 'https://github.com/blakeblackshear/frigate',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'Community',
+          items: [
+            {
+              label: 'GitHub',
+              href: 'https://github.com/blakeblackshear/frigate',
+            },
+            {
+              label: 'Discussions',
+              href: 'https://github.com/blakeblackshear/frigate/discussions',
+            },
+          ],
+        },
+      ],
+      copyright: `Copyright © ${new Date().getFullYear()} Blake Blackshear`,
+    },
+  },
+  plugins: [
+    path.resolve(__dirname, 'plugins', 'raw-loader'),
+    [
+      'docusaurus-plugin-openapi-docs',
+      {
+        id: 'openapi',
+        docsPluginId: 'classic', // configured for preset-classic
+        config: {
+          frigateApi: {
+            specPath: 'static/frigate-api.yaml',
+            outputDir: 'docs/integrations/api',
+            sidebarOptions: {
+              groupPathsBy: 'tag',
+              categoryLinkSource: 'tag',
+            },
+            hideSendButton: false,
+            showSchemas: true,
+          } satisfies OpenApiPlugin.Options,
+        },
+      },
+    ]
+  ] as PluginConfig[],
+  presets: [
+    [
+      'classic',
+      {
+        docs: {
+          routeBasePath: '/',
+          sidebarPath: './sidebars.ts',
+          // Please change this to your repo.
+          editUrl: 'https://github.com/blakeblackshear/frigate/edit/master/docs/',
+          sidebarCollapsible: true,
```

Comment:
This change makes it more dense and easier to read **IMO**

<img width="340" alt="image" src="https://github.com/user-attachments/assets/23f9f40e-eb39-409d-ab99-630432377ed1">

If other maintainers don't want we can revert it. Look at the video I've left in the description to see how it looks like

Reply:
IMO It's too many clicks, to get the same link for configuration support takes a lot more time

Reply:
I did it mainly because of the API documents. Now, there is a lot of scroll movement. 

https://github.com/user-attachments/assets/7ed22d48-1cf5-49da-9471-67e0243c2e31

I'll try to see If I can collapse only the HTTP API

Reply:
Was able to solve it by only collapsing the sidebar for the HTTP API

![image](https://github.com/user-attachments/assets/88435929-0e7f-4c90-b364-e983e6387647)